packages/frontend/src/scripts/configureTopBars.ts

```diff
@@ -0,0 +1,120 @@
+interface TopBarVariantData {
+  variant: 'gitcoin' | 'l2warsaw'
+  lastBannerChangeTime: number
+}
+
+const localStorageKey = 'topBarVariantData'
+
+const gitcoinStartDate = new Date('2023-08-15T12:00:00.000Z')
+const gitcoinEndDate = new Date('2023-08-29T12:00:00.000Z')
+
+export function configureTopBars() {
+  const variant = getBannerVariant()
+
+  const gitcoinBanner = document.querySelector('.TopBar-Gitcoin')
+  const l2warsawBanner = document.querySelector('.TopBar-L2Warsaw')
+
+  if (variant === 'gitcoin') {
+    gitcoinBanner?.classList.remove('hidden')
+    l2warsawBanner?.classList.add('hidden')
+  }
+
+  if (variant === 'l2warsaw') {
+    gitcoinBanner?.classList.add('hidden')
+    l2warsawBanner?.classList.remove('hidden')
+  }
+}
+
+//
+//     August 2023
+// Mo Tu We Th Fr Sa Su
+//     1  2  3  4  5  6
+//  7  8  9 10 11 12 13
+// 14 15 16 17 18 19 20
+// 21 22 23 24 25 26 27
+// 28 29 30 31
+//
+// NOTE(radomski): Gitcoin round starts on the 15th and ends on the 29th. On
+// 15th, 27th, 28th, and 29th no matter what the local storage says, display
+// the gitcoin banner. On those days we do not need to write anything into the
+// local storage since it's always going to be gitcoin. On the first occurance
```

Comment:
```suggestion
// local storage since it's always going to be gitcoin. On the first occurrence
```

Reply:
Sooo, that's the longer version of what miszke was talking about. Appreciate the comment anyway 😅